src/index.js

```diff
@@ -607,6 +607,32 @@ class Enmap extends Map {
     return this.has(key, path);
   }
 
+  /**
+   * Performs Array.includes() on a certain enmap value. Works similar to
+   * [Array.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes).
+   * @param {string|number} key Required. The key of the array to check the value of.
```

Comment:
The key of the array or object including the array, right? (hence the path)

Reply:
In the context of Enmap, `key` always refers to the "main" key of the data stored, never anything in the value stored.

Reply:
I'm aware of that, I meant that the description of this parameter is not fully correct. As far as I know the documentation is created with this, so it might be an issue for some. Perhaps "key of the `element`" would be better?

I don't mean to be rude but was just giving two cents. This PR was made after my question on Discord, exactly because the docs of another function (`hasProp`) was unclear to me. E.g., "Returns whether or not the property exists within an object or array value in enmap.", while you can't use it to check for an array value.

Anyway, thanks for adding this @DennisV2809!